.github/workflows/rust.yml

```
@@ -218,3 +218,20 @@ jobs:
       - name: Stop docker-compose
         working-directory: ./docker
         run: docker-compose down
+
+  docs:
+    name: cargo-doc
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+
+      - name: Check the building of docs
+        working-directory: ./rust
+        run: cargo doc --all-features --color always
```

this should fail on our current documentation if we don't use the nightly toolchain, because auto link generation is not yet stable.

```suggestion
        run: cargo +nightly doc --all-features --color always
```